.github/workflows/test.yml

```
@@ -0,0 +1,82 @@
+name: Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up env
+      run: |
+        sudo apt update
+        sudo apt install -y cppcheck
+  
+    - name: Install buildifier
+      run: |
+        sudo wget -q https://github.com/bazelbuild/buildtools/releases/download/0.29.0/buildifier -O /usr/local/bin/buildifier
+        sudo chmod +x /usr/local/bin/buildifier
+
+    - name: Coding style check
+      run: ./tools/ci_format_check common utils
+
+    - name: Buildifier check
+      run: ./tools/ci_buildifier_check
+
+    - name: Static analysis check
+      run: cppcheck --force --error-exitcode=1 -q common utils
+
+    - name: TE=1 bazel test opt ASAN
+      run: bazel test --build_tests_only -c opt --test_output=all --config asan --define trit_encoding=1 -- //... -//mobile/ios/...
```

there is no reason to test Android and iOS on Ubuntu, please change other test cases as well. 

```suggestion
      run: bazel test --build_tests_only -c opt --test_output=all --config asan --define trit_encoding=1 -- //... -//mobile/...
```